jetty-proxy/src/test/java/org/eclipse/jetty/proxy/AsyncMiddleManServletTest.java

```diff
@@ -508,46 +508,52 @@ protected ContentTransformer newClientRequestContentTransformer(HttpServletReque
                     throw new NullPointerException("explicitly_thrown_by_test");
                 };
             }
-        });
+        };
+        startServer(new EchoHttpServlet());
```

Comment:
Is the changed order of starting significant?  If not can we keep the order to minimise changes

Reply:
This specific change was to take StacklessLogging behavior from the scope of the entire Test class, to specific methods where it was appropriate.
Allowing stacks to be produced in the other cases where knowing details about the failure is important.

I'll review the start order.

Reply:
Yes please restore the inlining of `AsyncMiddleManServlet`.

Reply:
Restored.